internal/provider/resource_gitlab_group_project_file_template.go

```diff
@@ -0,0 +1,116 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	gitlab "github.com/xanzy/go-gitlab"
+	"log"
+)
+
+var _ = registerResource("gitlab_group_project_file_template", func() *schema.Resource {
+	return &schema.Resource{
+		Description: `The ` + "`gitlab_group_project_file_template`" + ` resource allows setting a project from which
+custom file templates will be loaded. The project selected must be a direct child of the group identified.
+For more information about which file types are available as templates, view 
+[GitLab's documentation](https://docs.gitlab.com/ee/user/admin_area/settings/instance_template_repository.html#supported-file-types-and-locations)
+
+-> This resource requires a GitLab Enterprise instance with a Premium license.
+
+**Upstream API**: [GitLab REST API docs](https://docs.gitlab.com/ee/api/groups.html#update-group)`,
+
+		// Since this resource updates an in-place resource, the update method is the same as the create method
+		CreateContext: resourceGitLabGroupProjectFileTemplateCreateOrUpdate,
+		UpdateContext: resourceGitLabGroupProjectFileTemplateCreateOrUpdate,
+		ReadContext:   resourceGitLabGroupProjectFileTemplateRead,
+		DeleteContext: resourceGitLabGroupProjectFileTemplateDelete,
```

Comment:
What about the importer 🤔  ?

Reply:
@timofurrer  - I considered this, but importing doesn't make much sense in this particular case. Importing tends to be used to manage a pre-existing resource, but there isn't really a resource being managed here, so there isn't much to import. If you wanted to overwrite what was there, you'd just... include the resource block, and the first time it ran it would overwrite the existing value.

We could certainly add one, it just seems like it's code we need to maintain that doesn't actually provide value :-/